### ISSUE TITLE
TypeError: unsupported operand type(s) for |: 'ModelMetaclass' and 'NoneType'

### ISSUE THREAD

**Issue by Artucuno** [Hmiku8338/e621#2]
Hey, I'm trying to get e926 to work and I am getting.
```py
>>> from e621 import E926
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\User\AppData\Local\Programs\Python\Python39\lib\site-packages\e621\__init__.py", line 1, in <module>
    from .api import E621, E926
  File "C:\Users\User\AppData\Local\Programs\Python\Python39\lib\site-packages\e621\api.py", line 3, in <module>
    from . import endpoints
  File "C:\Users\User\AppData\Local\Programs\Python\Python39\lib\site-packages\e621\endpoints.py", line 28, in <module>
    from .models import (
  File "C:\Users\User\AppData\Local\Programs\Python\Python39\lib\site-packages\e621\models.py", line 405, in <module>
    class Post(BaseModel):
  File "pydantic\main.py", line 188, in pydantic.main.ModelMetaclass.__new__
  File "pydantic\typing.py", line 419, in pydantic.typing.resolve_annotations
    def Union(self, parameters):
  File "C:\Users\User\AppData\Local\Programs\Python\Python39\lib\typing.py", line 277, in _eval_type
    return t._evaluate(globalns, localns, recursive_guard)
  File "C:\Users\User\AppData\Local\Programs\Python\Python39\lib\typing.py", line 533, in _evaluate
    eval(self.__forward_code__, globalns, localns),
  File "<string>", line 1, in <module>
TypeError: unsupported operand type(s) for |: 'ModelMetaclass' and 'NoneType'
```

Thanks

**Comment by Artucuno** [comment]
This also happens for e621

**Comment by Hmiku8338** [comment]
On it.

**Comment by Hmiku8338** [comment]
Fixed in 1.0.4
